devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -245,6 +245,11 @@ private Map getButtonResult(buttonState, buttonNumber = 1) {
 
 def installed() {
     initialize()
+
+    // Initialize default states
+    device.currentValue("numberOfButtons")?.times {
+        sendEvent(name: "button", value: "pushed", data: [buttonNumber: it+1])
```

Comment:
Probably want to add `displayed: false`

Reply:
Should I add that to the "numberOfButton" events, too? Probably a good idea while I am here.

Reply:
yeah i'd say so

Reply:
Done!